### DISCUSSION TITLE
not found in gene family

### DISCUSSION THREAD

**Discussion by miaommpt** [hahnlab/CAFE5#182] [Q&A] [answered]
Hi everyone,

While I'm using CAFE, it always showed RA was not found in gene family 1. or it cannot find another species Allochromatium_humboldtianum not found in gene family 1. I tried to search hidden character but did not find any. Also I tried to change name of the species, not working. 

Please help if you have any suggestions. I attached my files here.

Thanks a lot,

Hailey
[cafe_input_latest.txt](https://github.com/user-attachments/files/15831864/cafe_input_latest.txt)
[tree.txt](https://github.com/user-attachments/files/15831869/tree.txt)

**Comment by benfulton** [comment, marked as answer]
You have your family file transposed - the species should be in the columns and the gene families in the rows. I've attached a corrected file, but I don't think you will be able to use CAFE for this as your data set is mostly zeroes with a few ones. CAFE will probably need more gene families to be able to conduct an analysis.

[miaommpt_families.txt](https://github.com/user-attachments/files/15838442/miaommpt_families.txt)
